README.md

```diff
@@ -28,4 +28,6 @@ npm install -g truffle ganache-cli
 
 truffle compile
 truffle migrate
-```
\ No newline at end of file
+```
+
+MIT License
```

Comment:
Consider adding a brief explanation or description for the MIT License, to provide more context for users and contributors.